### DISCUSSION TITLE
Are new contributions accepted?

### DISCUSSION THREAD

**Discussion by Ishaanjolly** [google/or-tools#4387] [General]
I just thought of asking if new contributors are encouraged? If so, are there any `good first issue` labels for the newbies?

**Comment by lperron** [comment]
yes. The issues are not trivial though.

The simplest would be to contribute examples about a problem you like, or about some techniques you are working on.

You can also reviews code and docs and propose improvements.

**Comment by Ishaanjolly** [comment]
sure, i am happy to write a couple of examples within notebooks and python, if that's alright?

**Comment by lperron** [comment]
yes, nice visualizations in colab are a big +.
Laurent Perron | Operations Research | ***@***.*** | (33) 1 42 68 53
00



Le mer. 25 sept. 2024 à 13:01, Ishaan Jolly ***@***.***> a
écrit :
